packages/contract-addresses/CHANGELOG.json

```diff
@@ -1,4 +1,12 @@
 [
+    {
+        "version": "2.3.2",
+        "changes": [
+            {
+                "note": "Redeploy Coordinator contract"
+            }
```

Comment:
PR number?

Reply:
adding!

is patch version bump OK or should we use minor?